src/containers/DetailContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+import QueenDetail from '../components/Detail/QueenDetail.jsx';
+import { fetchQueenById } from '../services/fetchQueenById.js';
+
+const QueenPicker = ({ match }) => {
```

Comment:
Don't forget to add PropTypes for this prop. You can write something like this when dealing with nested objects:
```
QueenPicker.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      id: PropTypes.string.isRequired
    }).isRequired
  }).isRequired
};
```